packages/utils/telemetry-utils/src/test/types/validateTelemetryUtilsPrevious.ts

```diff
@@ -600,6 +600,7 @@ declare function get_old_ClassDeclaration_LoggingError():
 declare function use_current_ClassDeclaration_LoggingError(
     use: TypeOnly<current.LoggingError>);
 use_current_ClassDeclaration_LoggingError(
+    // @ts-expect-error - Not sure why this isn't getting generated
```

Comment:
@anthony-murphy - I tried to get the package.json markup right but maybe I messed it up, this wasn't getting generated so I threw this in to unblock myself in local build.  Can you take a look and see if I have a typo, or if you know what might be going on?  Thanks!

Reply:
Maybe it needs to be .2002 or something?

Reply:
did you run `npm run testtypes:gen` from the root? that will regen everything.

Reply:
@anthony-murphy ping - I'm not seeing the typetests scripts in the package.json, and if I run `fluid-type-validator -d .` it generates validate0.58.2000.ts, as opposed to validateTelemetryUtilsPrevious.ts.  So what's the workflow here...?

Reply:
oops didn't see your reply